releasePipeline/05-deploy-maven-galasa.sh

```diff
@@ -0,0 +1,208 @@
+# echo "Running from directory ${BASEDIR}"
+export ORIGINAL_DIR=$(pwd)
+# cd "${BASEDIR}"
+
+cd "${BASEDIR}/.."
+WORKSPACE_DIR=$(pwd)
+
+
+#-----------------------------------------------------------------------------------------                   
+#
+# Set Colors
+#
+#-----------------------------------------------------------------------------------------                   
+bold=$(tput bold)
+underline=$(tput sgr 0 1)
+reset=$(tput sgr0)
+red=$(tput setaf 1)
+green=$(tput setaf 76)
+white=$(tput setaf 7)
+tan=$(tput setaf 202)
+blue=$(tput setaf 25)
+
+#-----------------------------------------------------------------------------------------                   
+#
+# Headers and Logging
+#
+#-----------------------------------------------------------------------------------------                   
+underline() { printf "${underline}${bold}%s${reset}\n" "$@"
+}
+h1() { printf "\n${underline}${bold}${blue}%s${reset}\n" "$@"
+}
+h2() { printf "\n${underline}${bold}${white}%s${reset}\n" "$@"
+}
+debug() { printf "${white}%s${reset}\n" "$@"
+}
+info() { printf "${white}➜ %s${reset}\n" "$@"
+}
+success() { printf "${green}✔ %s${reset}\n" "$@"
+}
+error() { printf "${red}✖ %s${reset}\n" "$@"
+}
+warn() { printf "${tan}➜ %s${reset}\n" "$@"
+}
+bold() { printf "${bold}%s${reset}\n" "$@"
+}
+note() { printf "\n${underline}${bold}${blue}Note:${reset} ${blue}%s${reset}\n" "$@"
+}
+
+
+#-----------------------------------------------------------------------------------------                   
+# Main logic.
+#-----------------------------------------------------------------------------------------                   
+
+mkdir -p temp
+
+function ask_user_for_release_type {
+    PS3="Select the type of release process please: "
+    select lng in release pre-release
+    do
+        case $lng in
+            "release")
+                export release_type="release"
+                break
+                ;;
+            "pre-release")
+                export release_type="prerelease"
+                break
+                ;;
+            *)
+            echo "Unrecognised input."
+            exit 1
+            ;;
+        esac
+    done
+    echo "Chosen type of release process: ${release_type}"
+}
+
+function ask_user_for_release_version {
+    h1 "Please type the version of this release..."
+
+    read -p "Please enter the version number: " version
+
+    echo "You are doing a release for version ${version}"
+}
+
+
+function set_kubernetes_context {
+    namespace="galasa-build"
+    h1 "Setting the kubernetes context to be cicsk8s, using namespace ${namespace}"
+    kubectl config set-context cicsk8s --namespace=${namespace}
+    rc=$?
+    if [[ "${rc}" != "0" ]]; then 
+        error "Failed. rc=${rc}"
+        exit 1
+    fi
+    success "Kubernetes context set to cicsk8s using namespace ${namespace}"
+}
+
+
+
+function deploy_maven_artifacts {
+
+    h1 "Deploying all artifacts in github called ${release_type}"
```

Comment:
The artefacts are not being deployed from GitHub here - can you change this to 'Deploying all Galasa artefacts at version ${version}' please?